package.json

```diff
@@ -537,6 +537,12 @@
             ],
```

Comment:
I think we need to add the other commands to this enum.

Reply:
(or are you doing that in a separate PR?)

Reply:
As they are not ready for users to use, I didn't add them here, I only added them to the code for our future development.
But I added them after your comment. Let me know if you want them to stay.

Reply:
We can have them in the code, but not `package.json` or else people will get IntelliSense for a feature that is not available yet.

Reply:
So I Will revert this change.